tests/UI/Http/Rest/Controller/JsonApiTestCase.php

```diff
@@ -70,7 +70,10 @@ protected function auth(string $username = self::DEFAULT_EMAIL, string $password
             '_password' => $password ?: self::DEFAULT_PASS,
         ]);
 
-        $response = json_decode($this->cli->getResponse()->getContent(), true);
+        /** @var string $content */
+        $content = $this->cli->getResponse()->getContent();
+
+        $response = json_decode($content, true);
```

Comment:
Is this really needed?

Reply:
@Lutacon Thanks for review, appreciated that.

Yes this is necessary because `getContent()` return string, so we need decode this to get token.